apix_documentation_swagger_v3.yaml

```diff
@@ -204,10 +204,15 @@ info:
         response has been received yet
       - __on_hold__ This status has to be manually set; transaction will not
         be processed until it is changed back to `initialized`
-      - __confirmed__ Payment was successful
+      - __confirmed__ Payment was successful, in the case of check payment,
+        it means that the check was processed for delivery
       - __rejected__ There was a problem with the payment; you can get more
         information by checking the __error_code__ and __error_message__ in
         the transaction’s response.
+      - __canceling__ Immediate status response, after cancel attempt, of check payment
+      - __canceled__ Confirmation of payment cancelation via check
+      - __linked__ Check payment is in process, is the next status after `initialized `status
```

Comment:
Check payment is in process; it is the next status after the `initialized` status